### ISSUE TITLE
IGC parser fails

### ISSUE THREAD

**Issue by dspreitz** [Turbo87/aeroscore#29]
The IGC parser fails when reading IGC files provided from the Bayreuth contest( Open class, 30. May).
Example IGC files provided as attachment.
[Bayreuth_2018-05-30_Open_only_official_IGCs.tar.gz](https://github.com/Turbo87/aeroscore/files/2055780/Bayreuth_2018-05-30_Open_only_official_IGCs.tar.gz)

```
ogn@data-ogn:~/aeroscore$ ts-node examples/calc-ranking.ts fixtures/Bayreuth/task_open fixtures/Bayreuth/2018-05-30/
Error: Invalid task declaration at line 16: C3005181604053005180001-2
    at IGCParser.parseTask (/home/ogn/aeroscore/node_modules/igc-parser/index.ts:321:13)
    at IGCParser.processTaskLine (/home/ogn/aeroscore/node_modules/igc-parser/index.ts:312:32)
    at IGCParser.processLine (/home/ogn/aeroscore/node_modules/igc-parser/index.ts:196:12)
    at Function.IGCParser.parse (/home/ogn/aeroscore/node_modules/igc-parser/index.ts:155:14)
    at readFlightFromString (/home/ogn/aeroscore/src/read-flight.ts:14:20)
    at Object.readFlight (/home/ogn/aeroscore/src/read-flight.ts:23:10)
    at /home/ogn/aeroscore/examples/calc-ranking.ts:40:39
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/home/ogn/aeroscore/examples/calc-ranking.ts:35:4)
    at Module._compile (module.js:652:30)
```

**Comment by Turbo87** [comment]
the issue in this case is https://github.com/Turbo87/igc-parser/issues/13 and has been fixed by https://github.com/Turbo87/igc-parser/pull/14 which was released in v0.3.2.

**Comment by dspreitz** [comment]
I tested it with some recent IGC files from the Bayreuth competition and confirm it is now working. Thank you for the fix.
